spec/unit/packages_inspector_spec.rb

```diff
@@ -31,68 +31,210 @@
   }
   let(:packages_inspector) { PackagesInspector.new(system, description) }
 
-  let(:package_example) { <<EOF
+  context "rpm inspection" do
+    let(:package_example) { <<EOF
 zypper|1.9.16|22.2|x86_64|openSUSE|4a87f6b9ceae5d40a411fe52d0f17050$
 rpm|4.11.1|6.5.1|x86_64|openSUSE|7dfdd742a9b7d60c75bf4844d294716d$
 EOF
-  }
-  let(:expected_packages) {
-    PackagesScope.new(
-      [
-        Package.new(
-          name: "rpm",
-          version: "4.11.1",
-          release: "6.5.1",
-          arch: "x86_64",
-          vendor: "openSUSE",
-          checksum: "7dfdd742a9b7d60c75bf4844d294716d"
-        ),
-        Package.new(
-          name: "zypper",
-          version: "1.9.16",
-          release: "22.2",
-          arch: "x86_64",
-          vendor: "openSUSE",
-          checksum: "4a87f6b9ceae5d40a411fe52d0f17050"
-        )
-      ],
-      package_system: "rpm"
-    )
-  }
-  let(:rpm_command) {
-    ["rpm", "-qa", "--qf", "%{NAME}|%{VERSION}|%{RELEASE}|%{ARCH}|%{VENDOR}|%{FILEMD5S}$", stdout: :capture]
-  }
+    }
+    let(:expected_packages) {
+      PackagesScope.new(
+        [
+          Package.new(
+            name: "rpm",
+            version: "4.11.1",
+            release: "6.5.1",
+            arch: "x86_64",
+            vendor: "openSUSE",
+            checksum: "7dfdd742a9b7d60c75bf4844d294716d"
+          ),
+          Package.new(
+            name: "zypper",
+            version: "1.9.16",
+            release: "22.2",
+            arch: "x86_64",
+            vendor: "openSUSE",
+            checksum: "4a87f6b9ceae5d40a411fe52d0f17050"
+          )
+        ],
+        package_system: "rpm"
+      )
+    }
+    let(:rpm_command) {
+      ["rpm", "-qa", "--qf", "%{NAME}|%{VERSION}|%{RELEASE}|%{ARCH}|%{VENDOR}|%{FILEMD5S}$", stdout: :capture]
+    }
 
-  def inspect_data(host = "myhost", data = package_example)
-    expect(system).to receive(:run_command) { data }
+    before(:each) do
+      expect(system).to receive(:has_command?).with("rpm").and_return(true)
+    end
 
-    packages_inspector.inspect(filter)
-    description.packages
-  end
+    def inspect_data(host = "myhost", data = package_example)
```

Comment:
Unused method argument - `host`. If it's necessary, use `_` or `_host` as an argument name to indicate that it won't be used.